follow-ups.md

```diff
@@ -30,10 +31,14 @@
 - Always assign a number to `_vnodeId` (not null, use 0 to clear?)
 - Consider converting Fragment (and other special nodes, e.g. root, Portal,
   context?) to be numbers instead of functions with special diff handling
-  - One benefit of using numbers for Roots/Portals is that a portal could be
-    created with a `null` \_parentDOM prop and still be marked as a root type.
-    Particularly, this is useful for Suspense and would remove the need for the
-    call to `getParentDom` in `Suspense.render`
+
+  One benefit of using numbers for Roots/Portals is that a portal could be
+  created with a `null` \_parentDOM prop and still be marked as a root type.
+  Particularly, this is useful for Suspense and would remove the need for the
+  call to `getParentDom` in `Suspense.render`
```

Comment:
This makes me extremely excited for devtools, because in 10.x we need to pass function references around to detect if we're dealing with `Fragment`s or other special components. Would be much easier if it's just a serializable value like a number 👍